### DISCUSSION TITLE
[Enhancement] Entering characters with a Macron

### DISCUSSION THREAD

**Discussion by Cityyankee24** [neanes/neanes#524] [Ideas]
To be able to enter a character such as e or ē (ALT + 0275). Mostly for scenarios of pronouncing 'the' or 'thē' in English chant. At the moment, I'd have to enter it in a txt file then copy paste over. Unless I am missing something something, let me know! Thank you!

**Comment by danielgarthur** [comment]
Added to[ 0.4.17](https://github.com/neanes/neanes/releases/tag/v0.4.17).

In the lyrics toolbar, a button has been added for quicker entry of the ē, as well as some other Greek ligatures.

<img width="361" alt="image" src="https://github.com/neanes/neanes/assets/4132779/e2ed155a-2650-4f87-bbf7-44a707103ece">

As far as I know, only MacOS provides a convenient way to enter ē. If you press and hold <kbd>e</kbd>, then a little menu pops up with several different accented variations, including the macron accent. But Windows does not have this out of the box, although I'm sure there are probably apps for it.
